fdbserver/GetEncryptCipherKeys.actor.cpp

```diff
@@ -0,0 +1,256 @@
+/*
+ * GetCipherKeys.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbserver/GetEncryptCipherKeys.h"
+
+#include <boost/functional/hash.hpp>
+
+namespace {
+
+Optional<UID> getEncryptKeyProxyId(const Reference<AsyncVar<ServerDBInfo> const>& db) {
```

Comment:
a nit: any reason this is `Optional`?

Reply:
In case db->encryptKeyProxy is empty, this function return empty.